lib/__tests__/_createInsightsClient.test.ts

```diff
@@ -0,0 +1,32 @@
+import { createInsightsClient } from "../_createInsightsClient";
+
+describe("createInsightsClient", () => {
+  beforeEach(() => {
+    window.AlgoliaAnalyticsObject = undefined;
+  });
+
+  it("should return a function", () => {
+    expect(typeof createInsightsClient(() => {})).toBe("function");
+  });
```

Comment:
Why do you need to test this? Isn't it covered already?

Reply:
there was no test for this yet, so I wanted to add at least "hey it works"